### DISCUSSION TITLE
[Feature] Add Key-value attributes/properties

### DISCUSSION THREAD

**Discussion by MennoK** [wkentaro/labelme#1112] [Ideas / Feature requests]
**Is your feature request related to a problem? Please describe.**
For a data set which is  going to be used for instance segmentation, I want to add for each annotation certain properties with non-discrete values. For example I have dataset of objects and I want to add a mass attribute and add a ground-truth mass (which is a floating number) to the annotated object. In this case the current labelflags doesn't suffice in this case.

**Describe the solution you'd like**
Each time you have to choose a label for the annotation, you also have the option to select a certain attribute, and for a selected attribute you have to fill in a number, string or whatever.

**Comment by gladcolor** [comment]
I am working on this. And it would be great if you are working on this feature!
